packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -571,6 +571,30 @@ class DDSCarousel extends HostListenerMixin(StableSelectorMixin(LitElement)) {
           (elems.length - 1);
   }
 
+  private _updateContentsPosition(changedProperties) {
+    // The calculation of the contents position is base on start,
+    // _contentsBaseWidth, _gap and pageSize. If none of them changed, we've
+    // got nothing to do.
+    if (
+      !changedProperties.has('start') &&
+      !changedProperties.has('_contentsBaseWidth') &&
+      !changedProperties.has('_gap') &&
+      !changedProperties.has('pageSize')
+    ) {
+      return;
+    }
```

Comment:
> If none of them changed...

Does that mean these should be OR operators?

Reply:
Nope. Each operand is negated, so it reads "if start hasn't changed, and _contentBaseWidth hasn't changed and _gap hasn't change and pageSize hasn't changed then we've got nothing to do, return.